### ISSUE TITLE
New want: I want devtools to be able to flag accessibility errors in the console the same way JavaScript errors are.

### ISSUE THREAD

**Issue by aarongustafson** [WebWeWant/webwewant.fyi#71]
---
title: I want devtools to be able to flag accessibility errors in the console the same way JavaScript errors are.
date: 2020-09-16T20:51:17.205Z
submitter: PRIVATE
number: 5f627ac5a593f930c872566f
tags: []
---

The lack of dynamic accessibility checks means that accessibility efforts take a back seat during product development requiring special sprints to address any issues. Our team built an in-house checker using axe-core that shows any errors in the UI but that makes the local development very sluggish.

**Comment by aarongustafson** [comment]
Duplicates https://webwewant.fyi/wants/11/
